lib/info.js

```diff
@@ -330,7 +330,19 @@ const parseFormats = player_response => {
  * @param {Object} options
  * @returns {Promise<Object>}
  */
+const { version } = require('../package.json');
+let lastUpdateCheck = Date.now();
 exports.getInfo = async(id, options) => {
+  update = process.env.UPDATE;
```

Comment:
how about `process.env.YTDL_NO_UPDATE`? just `UPDATE` may be too common and used by other programs